tutorials/proto_rename/BUILD

```diff
@@ -1,13 +1,13 @@
 package(default_visibility = ["//visibility:private"])
 
 proto_library(
-    name = "person_proto",
-    srcs = ["person.proto"],
+    name = "persons_proto",
```

Comment:
Let's keep the old name `person.proto` even though it has a list inside.

Reply:
Done.